inc/saiqueue.h

```diff
@@ -41,12 +41,18 @@ typedef enum _sai_queue_type_t
     /** H/w Queue for all types of traffic */
     SAI_QUEUE_TYPE_ALL = 0x00000000,
 
-    /** H/w Unicast Queue */
+    /** H/w Egress Unicast Queue */
     SAI_QUEUE_TYPE_UNICAST = 0x00000001,
 
-    /** H/w Multicast (Broadcast, Unknown unicast, Multicast) Queue */
+    /** H/w Multicast Egress (Broadcast, Unknown unicast, Multicast) Queue */
     SAI_QUEUE_TYPE_MULTICAST = 0x00000002,
 
+    /** H/w Virtual Output Queue (VOQ). This queue is ingress unicast queue */
```

Comment:
please add comment Multicast VOQ is not supported as of now

Reply:
Multicast VoQ is out of scope in this initial VoQ system PR but shall we add the attribute now?

Reply:
Will add comment.

Reply:
Will add the attribute for multicast VOQ

Reply:
Added SAI_QUEUE_TYPE_MULTICAST_VOQ